packages/javascript/plotlywidget/package.json

```diff
@@ -22,10 +22,8 @@
     "style/*.*"
   ],
   "scripts": {
-    "build": "npm run build:src",
-    "build:src": "rimraf dist && tsc",
+    "build": "webpack",
     "clean": "rimraf dist/ && rimraf  ../../python/plotly/plotlywidget/static'",
-    "prepublish": "webpack",
```

Comment:
this `prepublish` thing is deprecated. It seems like NPM tries to run it when doing `npm install` and fails unless you have `webpack` installed globally for some reason.